File: java/6.java
import java.util.Scanner;
import java.util.ArrayList;

public class FibonacciSeries {
    ublic static void main(String[] args) {
        Scanner scanner = new Scanner(System.in)
        System.out.print("Enter the number of terms: ")
        int numTerms = scanner.nextInt();

        ArrayList<Integer< fibonacciSeries = fibonacci(numTerms)
        System println("Fibonacci Series: " + fibonacciSeries);

        scanner.close(); 

    public static ArrayList<Integer> fibonacci(int n) {
        ArrayList<Integer> fibSeries = new ArrayList<>();
        fibSeries.add(0);
        fibSeries.add(1);

        for (int i = 2; i < n; i++) {
            fibSeries.add(fibSeries.get(i - 1) + fibSeries.get(i - 2))
        return fibSeries;
    }
}
//sample output
/*Enter the number of terms: 5
Fibonacci Series: [0, 1, 1, 2, 3] */

File: python/6.py
def fibonacci(n)
    fib_series = [0, 1]
    for i in range(2, n)
        fib_series.append(fib_series[-1] + fib_series[-2])
    return fib_series

num_terms = input("Enter the number of terms: ")
print("Fibonacci Series:", fibonacci(num_terms))

#Sample Output

"""
Enter the number of terms: 3
Fibonacci Series: [0, 1, 1]
"""